configuration.json

```diff
@@ -2,14 +2,111 @@
   "videoRecord": false,
```

Comment:
General comment: maybe it makes sense to throw an exception in the void methods and return null or empty values in non void methods?

Reply:
Well... In most cases null or empty values are not expected return values, so they have to be handled as error cases anyways, so this change will not make much difference, but we can discuss it with the team.